src/compiler/scala/tools/nsc/transform/UnCurry.scala

```diff
@@ -484,14 +486,14 @@ abstract class UnCurry extends InfoTransform
           case Apply(fn, args) =>
             // Read formals before `transform(fn)`, because UnCurry replaces T* by Seq[T] (see DesugaredParameterType).
             // The call to `transformArgs` below needs `formals` that still have varargs.
```

Comment:
The comment will confuse the next person to read it because of the name change. (I'm not quite sure about "formals" meaning types instead of symbols. Doesn't it mean definition as opposed to application?)